indico/modules/events/editing/models/editable.py

```diff
@@ -91,3 +92,11 @@ def event(self):
     def can_comment(self, user):
         return (self.event.can_manage(user, permission='paper_editing')
                 or self.contribution.is_user_associated(user, check_abstract=True))
+
+    @property
+    def review_conditions_valid(self):
+        review_conditions = editing_settings.get(self.event, 'review_conditions').values()
+        file_types = {file.file_type_id for file in self.revisions[-1].files}
+        if not file_types:
+            return True
+        return any(all(ft in file_types for ft in cond) for cond in review_conditions)
```

Comment:
looking at it again i think we can simplify this even more using set logic:

    return any(file_types >= set(cond) for cond in review_conditions)

nothing wrong with keeping the `all()` though - whatever you prefer

Reply:
Will go for the first one, Looks much simpler and readable.